api/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>fetch-and-store-tweets</artifactId>
+        <groupId>com.brandwatch.internship</groupId>
```

Comment:
should this still be `internship`? Have seen `ahmedanalytics somewhere else`

Reply:
It works but of course I sill need to change it :-)